### ISSUE TITLE
Users Changing Fortunes

### ISSUE THREAD

**Issue by AlbertXDing** [cse110-sp23-group1/Origami-Fortune-Teller#1]
The user should have the option to change these by clicking on any box. This will create a new overlaying text box where the user can submit their new fortune.

**Comment by adarsh249** [comment]
Issue cse110-sp23-group1/Oragami-Fortune-Teller#2 must be done before doing this issue.

**Comment by phentos** [comment]
Why not edit the fortune in-line?

**Comment by phentos** [comment]
duplicate of #3
